packages/input-datepicker/src/LionInputDatepicker.js

```diff
@@ -280,17 +290,36 @@ export class LionInputDatepicker extends ScopedElementsMixin(OverlayMixin(LionIn
     `;
   }
 
+  _setupOverlayCtrl() {
+    super._setupOverlayCtrl();
+
+    this.__datepickerBeforeShow = () => {
+      this._overlayCtrl.updateConfig(this._defineOverlayConfig());
```

Comment:
Is it needed to always redefine the config?

Reply:
I'm not sure if checking a specific overlayCtrl config option would be "better"...

imho setting it every time is good enough for now

Reply:
Yeah, we do it once per switch on our extension layer.
Let's make a small improvement issue 👍